apps/login/package.json

```diff
@@ -5,7 +5,7 @@
     "dev": "next dev",
     "test": "concurrently --timings --kill-others-on-fail 'npm:test:unit' 'npm:test:integration'",
     "test:watch": "concurrently --kill-others 'npm:test:unit:watch' 'npm:test:integration:watch'",
-    "test:unit": "jest --config ./__test__/jest.config.ts",
+    "test:unit": "vitest",
```

Comment:
https://github.com/jestjs/jest/issues/9430 4 years later! By now, `test` is quite a pleasant experience and has proven in production.

Simple migration